extensions/resteasy-reactive/rest-client-reactive-kotlin-serialization/runtime/pom.xml

```diff
@@ -0,0 +1,158 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <artifactId>quarkus-rest-client-reactive-kotlin-serialization-parent</artifactId>
+        <groupId>io.quarkus</groupId>
+        <version>999-SNAPSHOT</version>
+        <relativePath>../pom.xml</relativePath>
+    </parent>
+    <artifactId>quarkus-rest-client-reactive-kotlin-serialization</artifactId>
+    <name>Quarkus - REST Client Reactive Kotlin Serialization</name>
+    <description>Kotlin serialization support for REST Client Reactive</description>
+
+    <dependencies>
+        <dependency>
+            <groupId>io.quarkus</groupId>
+            <artifactId>quarkus-rest-client-reactive</artifactId>
+            <exclusions>
+                <exclusion>
+                    <groupId>io.quarkus</groupId>
+                    <artifactId>quarkus-jackson</artifactId>
+                </exclusion>
+                <exclusion>
+                    <groupId>com.fasterxml.jackson.core</groupId>
+                    <artifactId>jackson-core</artifactId>
+                </exclusion>
+            </exclusions>
+        </dependency>
+        <dependency>
+            <groupId>io.quarkus</groupId>
+            <artifactId>quarkus-resteasy-reactive-kotlin-serialization-common</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>io.quarkus</groupId>
+            <artifactId>quarkus-jsonb</artifactId>
```

Comment:
Do we need jsonb for some reason? We shouldn't for this...

Reply:
It's just the API and i think rest assured was the problem here.  let me try moving that to the test.  There are some oddities in how all this is wired, though, so it could very well be needed.  even the jackson stuff seems to be end up pulling it in.  iirc, without it i get CNFE errors.  will confirm now that I have a working codebase, though.

Reply:
yep.  totally not necessary.  they're gone now.

Reply:
👍🏼